src/fal/cli/dbt_runner.py

```diff
@@ -166,3 +165,76 @@ def _create_fal_result_file(target_path: str, run_index: int):
         shutil.copy(
             fal_run_result, os.path.join(target_path, f"fal_results_{run_index}.json")
         )
+
+
+# This is the Python implementation of the `dbt_run()` function, in which
+# we directly use dbt-core as a Python library. We don't run it directly
+# but rather use 'multiprocessing' to run it in a real system Process to
+# imitate the existing behavior of `dbt_run()` (in terms of performance).
+
+# dbt-core is currently using the spawn as its mulitprocessing context
+# so we'll mirror it.
+multiprocessing.set_start_method("spawn", force=True)
```

Comment:
It shouldn't affect threading at any way I can imagine (since this is only for multiprocessing), but I don't think anything prevents us from shifting this inside the `dbt_run_through_python()` function (will do like that).

Reply:
The threading is from `multiprocessing` too https://github.com/fal-ai/fal/blob/bb3ded1fddff3d5eb0a3917430b8217710e9f4ae/src/fal/run_scripts.py#L5-L6

Reply:
I know you moved it, at this point I'm asking to learn.

Reply:
Cool, cool, cool. Thanks!